debian/rules

```diff
@@ -12,4 +12,4 @@ override_dh_installsystemd:
 
 override_dh_auto_install:
 	python3 ./setup.py install --root=$(CURDIR)/debian/tmp --prefix=/usr --install-layout=deb
-
+	install -Dm 644 etc/dbus-service.yaml $(CURDIR)/debian/tmp/etc/python-validity/dbus-service.yaml
```

Comment:
IIRC, `.` (as well as `debian/tmp`) is a part of the search path  for installation candidates. So if you move/rename `etc/dbus-service.yaml` to `etc/python-validity/dbus-service.yaml` inside the project, this command will no longer be needed. Not 100% sure though.

Reply:
Tested, that works